app/src/main/java/org/httpsknighthacks/knighthacksandroid/HorizontalSectionCard_RecyclerViewAdapter.java

```diff
@@ -175,30 +216,65 @@ public int getItemCount() {
     public class ContentViewHolder extends RecyclerView.ViewHolder {
         CardView mCardView;
         ImageView mCardImage;
+        ImageView mCardMapEvent;
         TextView mCardTitle;
         TextView mCardSideSubtitle;
         TextView mCardSubtitle;
         TextView mCardTagSubtitle;
         TextView mCardBody;
         TextView mCardTimestamp;
         TextView mCardFooter;
+        String mTag;
 
         public static final int VIEW_TYPE = 1;
 
-        public ContentViewHolder(View itemView) {
+        public ContentViewHolder(final View itemView, String tag) {
             super(itemView);
 
             View gridLayout = itemView.findViewById(R.id.horizontal_section_grid_view);
 
             this.mCardView = itemView.findViewById(R.id.horizontal_section_card_view);
             this.mCardImage = itemView.findViewById(R.id.horizontal_section_card_image);
+            this.mCardMapEvent = mDialog.findViewById(R.id.mapImage);
             this.mCardTitle = itemView.findViewById(R.id.horizontal_section_card_title);
             this.mCardSideSubtitle = itemView.findViewById(R.id.horizontal_section_card_side_subtitle);
             this.mCardSubtitle = itemView.findViewById(R.id.horizontal_section_card_subtitle);
             this.mCardTagSubtitle = gridLayout.findViewById(R.id.horizontal_section_card_tag_subtitle);
             this.mCardBody = itemView.findViewById(R.id.horizontal_section_card_body);
             this.mCardTimestamp = itemView.findViewById(R.id.horizontal_section_card_timestamp);
             this.mCardFooter = itemView.findViewById(R.id.horizontal_section_card_footer);
+            this.mTag = tag;
```

Comment:
I don't see mTag being referenced anywhere below this line. Do we need to have this?